_data/events.yml

```diff
@@ -629,3 +629,12 @@
   end_time: 2018-03-11 17:00 
   url: https://fifthelephant.in/2018-mar-dl-bootcamp/
   blurb: Learn theory and practical concepts of building a deep learning solution in the space of computer vision and natural language processing.
+  
+  name: vidhi-bengaluru-book-launch
```

Comment:
Add a hyphen (-) and space before `name`. For example, look at line number 624.

Reply:
Thanks Akshay. :) New to this.